prototypes/recipe/logistic/robot.lua

```diff
@@ -45,7 +45,7 @@ data.raw.recipe["wooden-chest"].ingredients = {{"wood", 4}}
 --Base "iron-chest"
 data.raw.recipe["iron-chest"].category = "crafting"
 data.raw.recipe["iron-chest"].energy_required = 1
-data.raw.recipe["iron-chest"].ingredients = {{"forging-iron", 1}, {"shaft-1", 1}, {"iron-plate", 2}}
```

Comment:
I think, the iron chest using an iron shaft is problematic, because its an item, which is used so far only for crude items like slow belts. I think, iron chests have an end game use (f.e. train stations, to limit buffering). Personally, I would stay with steel-chafts here.

Reply:
I would have iron chests use iron parts, and leave the steel parts to steel chests.

Reply:
@jodokus31 i understand your point but for the time you unlock the tech for using steel parts you probably will not gonna use something made of iron (considering the progression of the build materials used along the game in xander mod), much less in the point when you have train stations.

Reply:
Its also ok for me, if its changed like that. If I prefer iron chests, i can produce those iron-shafts, too.